index.html

```diff
@@ -26,15 +26,15 @@ <h1 class="name">Hannah Zeng</h1>
           </div>
         </div>
 
-        <nav>
+        <div id="nav-bar">
```

Comment:
It would be better to use the semantic nav tag, as the assignment suggests, to improve accessibility and SEO optimization.